### PR TITLE
Update 0.10.0 docs & refactor docs structure

### PR DESCRIPTION
**Pull request by furiosamg**

### CODE REVIEW

**Comment by furiosamg** [comment]
i18n 추가하려고 했는데, jupyter-notebook 과 호환이 되지 않는 것 같아 미루려고 합니다.
그리고 구성을 전체적으로 바꾸려고 하는데요 [HF](https://huggingface.co/docs/transformers/index#contents) 의 문서 구성이 맘에 들어서요 비슷하게 바꾸려고 합니다.
예상 구성
- GET STARTED provides a quick tour of the library and installation instructions to get up and running.
- TUTORIALS are a great place to start if you’re a beginner. This section will help you gain the basic skills you need to start using the library.
- HOW-TO GUIDES show you how to achieve a specific goal, like finetuning a pretrained model for language modeling or how to write and share a custom model.
- ADVANCED 모델, 전후처리 구조, rust postprocess 등에 대해서 더 설명
- MODELS
  - 모델 리스트
- API describes all classes and functions:
  - API 문서

**Comment by furiosamg** [comment]
/release-doc/v0.10.0

**Comment by furiosa-infra** [comment]
## Pip freeze result for all
<details><summary>pip freeze result</summary><pre>
aiobotocore==2.5.4
aiofiles==23.2.1
aiohttp==3.8.5
aiohttp-retry==2.8.3
aioitertools==0.11.0
aiosignal==1.3.1
amqp==5.1.1
annotated-types==0.5.0
antlr4-python3-runtime==4.9.3
appdirs==1.4.4
asttokens==2.4.0
async-timeout==4.0.3
asyncssh==2.13.2
atpublic==4.0
attrs==23.1.0
backcall==0.2.0
billiard==4.1.0
boto3==1.28.17
botocore==1.31.17
celery==5.3.4
certifi==2023.7.22
cffi==1.15.1
charset-normalizer==3.2.0
click==8.1.7
click-didyoumean==0.3.0
click-plugins==1.1.1
click-repl==0.3.0
cmake==3.27.5
colorama==0.4.6
comm==0.1.4
configobj==5.0.8
contourpy==1.1.1
cryptography==41.0.4
cycler==0.11.0
Cython==3.0.2
debugpy==1.8.0
decorator==5.1.1
dictdiffer==0.9.0
diskcache==5.6.3
distro==1.8.0
dpath==2.1.6
dulwich==0.21.6
dvc==3.22.1
dvc-data==2.16.3
dvc-http==2.30.2
dvc-objects==1.0.1
dvc-render==0.6.0
dvc-s3==2.23.0
dvc-studio-client==0.15.0
dvc-task==0.3.0
exceptiongroup==1.1.3
executing==1.2.0
fastjsonschema==2.18.0
filelock==3.12.4
flatten-dict==0.4.2
flufl.lock==7.1.1
fonttools==4.42.1
frozenlist==1.4.0
fsspec==2023.9.2
funcy==2.0
furiosa-common==0.10.0
furiosa-models @ file:///app
furiosa-native-postprocess==0.9.0.dev0
furiosa-native-runtime==0.10.1
furiosa-quantizer==0.10.0
furiosa-quantizer-impl==0.10.0
furiosa-runtime==0.10.0
furiosa-tools==0.10.0
gitdb==4.0.10
GitPython==3.1.37
grandalf==0.8
hydra-core==1.3.2
idna==3.4
importlib-metadata==6.8.0
importlib-resources==6.1.0
iniconfig==2.0.0
ipykernel==6.25.2
ipython==8.15.0
iterative-telemetry==0.0.8
jedi==0.19.0
Jinja2==3.1.2
jmespath==1.0.1
jsonschema==4.19.1
jsonschema-specifications==2023.7.1
jupyter_client==8.3.1
jupyter_core==5.3.1
kiwisolver==1.4.5
kombu==5.3.2
lit==17.0.0rc4
markdown-it-py==3.0.0
MarkupSafe==2.1.3
matplotlib==3.8.0
matplotlib-inline==0.1.6
mdurl==0.1.2
mpmath==1.3.0
multidict==6.0.4
multipledispatch==1.0.0
nbclient==0.6.8
nbformat==5.9.2
nbmake==1.4.3
nest-asyncio==1.5.8
networkx==3.1
numpy==1.25.2
nvidia-cublas-cu11==11.10.3.66
nvidia-cuda-cupti-cu11==11.7.101
nvidia-cuda-nvrtc-cu11==11.7.99
nvidia-cuda-runtime-cu11==11.7.99
nvidia-cudnn-cu11==8.5.0.96
nvidia-cufft-cu11==10.9.0.58
nvidia-curand-cu11==10.2.10.91
nvidia-cusolver-cu11==11.4.0.1
nvidia-cusparse-cu11==11.7.4.91
nvidia-nccl-cu11==2.14.3
nvidia-nvtx-cu11==11.7.91
omegaconf==2.3.0
onnx==1.14.1
opencv-python-headless==4.8.0.76
orjson==3.9.7
packaging==23.1
pandas==2.0.3
parso==0.8.3
pathspec==0.11.2
pexpect==4.8.0
pickleshare==0.7.5
Pillow==10.0.1
platformdirs==3.10.0
pluggy==1.3.0
prompt-toolkit==3.0.39
protobuf==4.24.3
psutil==5.9.5
ptyprocess==0.7.0
pure-eval==0.2.2
py-cpuinfo==9.0.0
pyarrow==12.0.1
pycocotools==2.0.7
pycparser==2.21
pydantic==2.3.0
pydantic_core==2.6.3
pydot==1.4.2
pygit2==1.13.1
Pygments==2.16.1
pygtrie==2.5.0
pyparsing==3.1.1
pytest==7.4.2
pytest-asyncio==0.17.2
pytest-benchmark==4.0.0
python-dateutil==2.8.2
pytz==2023.3.post1
PyYAML==6.0.1
pyzmq==25.1.1
referencing==0.30.2
requests==2.31.0
rich==13.5.3
rpds-py==0.10.3
ruamel.yaml==0.17.32
ruamel.yaml.clib==0.2.7
s3fs==2023.9.2
s3transfer==0.6.2
scmrepo==1.3.1
shortuuid==1.0.11
shtab==1.6.4
six==1.16.0
smmap==5.0.1
sqltrie==0.7.0
stack-data==0.6.2
sympy==1.12
tabulate==0.9.0
tomli==2.0.1
tomlkit==0.12.1
torch==2.0.1
torchvision==0.15.2
tornado==6.3.3
tqdm==4.66.1
traitlets==5.10.0
triton==2.0.0
typing_extensions==4.8.0
tzdata==2023.3
urllib3==1.26.16
vine==5.0.0
voluptuous==0.13.1
wcwidth==0.2.6
wrapt==1.15.0
yarl==1.9.2
zc.lockfile==3.0.post1
zipp==3.17.0
</pre></details>

**Comment by furiosa-infra** [comment]
add3e67 nits: import early, packaging stuffs correctly
```

---------------------------------------------------------------------------------------------------------- benchmark: 9 tests ---------------------------------------------------------------------------------------------------------
Name (time in ms)                                                  Min                   Max                Mean             StdDev              Median                 IQR            Outliers       OPS            Rounds  Iterations
---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
test_mlcommons_resnet50_accuracy                                1.5000 (1.0)         63.0499 (1.43)       7.6127 (1.0)       5.9274 (1.57)       5.4743 (1.0)        2.1434 (1.75)    5725;7380  131.3587 (1.0)       50000           1
test_efficientnetb0_accuracy                                    4.9034 (3.27)       317.2269 (7.20)      13.0868 (1.72)      7.0704 (1.87)      12.3864 (2.26)       2.2392 (1.83)    1077;2899   76.4128 (0.58)      50000           1
test_mlcommons_ssd_mobilenet_with_native_rust_pp_accuracy       6.0923 (4.06)        44.0522 (1.0)       12.4073 (1.63)      5.2489 (1.39)       9.6013 (1.75)       5.3920 (4.40)      788;324   80.5976 (0.61)       5000           1
test_efficientnetv2s_accuracy                                   7.1539 (4.77)       256.9700 (5.83)      15.2360 (2.00)      6.8004 (1.80)      14.6593 (2.68)       2.6217 (2.14)    1002;1531   65.6341 (0.50)      50000           1
test_mlcommons_ssd_mobilenet_accuracy                           7.3518 (4.90)       240.3726 (5.46)      11.1492 (1.46)      3.7776 (1.0)       10.6000 (1.94)       1.2245 (1.0)       468;673   89.6923 (0.68)       5000           1
test_yolov5m_accuracy                                           8.9905 (5.99)       778.9908 (17.68)     17.0584 (2.24)     14.1673 (3.75)      13.2752 (2.43)       6.0691 (4.96)     723;1142   58.6222 (0.45)      10000           1
test_yolov5l_accuracy                                          11.8736 (7.92)       188.9924 (4.29)      16.5485 (2.17)      7.1420 (1.89)      15.2527 (2.79)       3.1159 (2.54)      222;383   60.4284 (0.46)      10000           1
test_mlcommons_ssd_resnet34_with_native_rust_pp_accuracy       42.3397 (28.23)      727.7901 (16.52)     51.2738 (6.74)     15.5134 (4.11)      50.7233 (9.27)       1.9495 (1.59)        6;205   19.5031 (0.15)       5000           1
test_mlcommons_ssd_resnet34_accuracy                          104.9737 (69.98)    1,395.5332 (31.68)    285.7867 (37.54)    99.7738 (26.41)    286.5471 (52.34)    104.1625 (85.06)     997;231    3.4991 (0.03)       5000           1
---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------

Legend:
  Outliers: 1 Standard Deviation from Mean; 1.5 IQR (InterQuartile Range) from 1st Quartile and 3rd Quartile.
  OPS: Operations Per Second, computed as 1 / Mean
```

**Comment by furiosa-infra** [comment]
## Pip freeze result for all
<details><summary>pip freeze result</summary><pre>
aiobotocore==2.5.4
aiofiles==23.2.1
aiohttp==3.8.5
aiohttp-retry==2.8.3
aioitertools==0.11.0
aiosignal==1.3.1
amqp==5.1.1
annotated-types==0.5.0
antlr4-python3-runtime==4.9.3
appdirs==1.4.4
asttokens==2.4.0
async-timeout==4.0.3
asyncssh==2.13.2
atpublic==4.0
attrs==23.1.0
backcall==0.2.0
billiard==4.1.0
boto3==1.28.17
botocore==1.31.17
celery==5.3.4
certifi==2023.7.22
cffi==1.15.1
charset-normalizer==3.2.0
click==8.1.7
click-didyoumean==0.3.0
click-plugins==1.1.1
click-repl==0.3.0
cmake==3.27.5
colorama==0.4.6
comm==0.1.4
configobj==5.0.8
contourpy==1.1.1
cryptography==41.0.4
cycler==0.11.0
Cython==3.0.2
debugpy==1.8.0
decorator==5.1.1
dictdiffer==0.9.0
diskcache==5.6.3
distro==1.8.0
dpath==2.1.6
dulwich==0.21.6
dvc==3.22.1
dvc-data==2.16.3
dvc-http==2.30.2
dvc-objects==1.0.1
dvc-render==0.6.0
dvc-s3==2.23.0
dvc-studio-client==0.15.0
dvc-task==0.3.0
exceptiongroup==1.1.3
executing==1.2.0
fastjsonschema==2.18.0
filelock==3.12.4
flatten-dict==0.4.2
flufl.lock==7.1.1
fonttools==4.42.1
frozenlist==1.4.0
fsspec==2023.9.2
funcy==2.0
furiosa-common==0.10.0
furiosa-models @ file:///app
furiosa-native-postprocess==0.9.0.dev0
furiosa-native-runtime==0.10.1
furiosa-quantizer==0.10.0
furiosa-quantizer-impl==0.10.0
furiosa-runtime==0.10.0
furiosa-tools==0.10.0
gitdb==4.0.10
GitPython==3.1.37
grandalf==0.8
hydra-core==1.3.2
idna==3.4
importlib-metadata==6.8.0
importlib-resources==6.1.0
iniconfig==2.0.0
ipykernel==6.25.2
ipython==8.15.0
iterative-telemetry==0.0.8
jedi==0.19.0
Jinja2==3.1.2
jmespath==1.0.1
jsonschema==4.19.1
jsonschema-specifications==2023.7.1
jupyter_client==8.3.1
jupyter_core==5.3.1
kiwisolver==1.4.5
kombu==5.3.2
lit==17.0.0rc4
markdown-it-py==3.0.0
MarkupSafe==2.1.3
matplotlib==3.8.0
matplotlib-inline==0.1.6
mdurl==0.1.2
mpmath==1.3.0
multidict==6.0.4
multipledispatch==1.0.0
nbclient==0.6.8
nbformat==5.9.2
nbmake==1.4.3
nest-asyncio==1.5.8
networkx==3.1
numpy==1.25.2
nvidia-cublas-cu11==11.10.3.66
nvidia-cuda-cupti-cu11==11.7.101
nvidia-cuda-nvrtc-cu11==11.7.99
nvidia-cuda-runtime-cu11==11.7.99
nvidia-cudnn-cu11==8.5.0.96
nvidia-cufft-cu11==10.9.0.58
nvidia-curand-cu11==10.2.10.91
nvidia-cusolver-cu11==11.4.0.1
nvidia-cusparse-cu11==11.7.4.91
nvidia-nccl-cu11==2.14.3
nvidia-nvtx-cu11==11.7.91
omegaconf==2.3.0
onnx==1.14.1
opencv-python-headless==4.8.0.76
orjson==3.9.7
packaging==23.1
pandas==2.0.3
parso==0.8.3
pathspec==0.11.2
pexpect==4.8.0
pickleshare==0.7.5
Pillow==10.0.1
platformdirs==3.10.0
pluggy==1.3.0
prompt-toolkit==3.0.39
protobuf==4.24.3
psutil==5.9.5
ptyprocess==0.7.0
pure-eval==0.2.2
py-cpuinfo==9.0.0
pyarrow==12.0.1
pycocotools==2.0.7
pycparser==2.21
pydantic==2.3.0
pydantic_core==2.6.3
pydot==1.4.2
pygit2==1.13.1
Pygments==2.16.1
pygtrie==2.5.0
pyparsing==3.1.1
pytest==7.4.2
pytest-asyncio==0.17.2
pytest-benchmark==4.0.0
python-dateutil==2.8.2
pytz==2023.3.post1
PyYAML==6.0.1
pyzmq==25.1.1
referencing==0.30.2
requests==2.31.0
rich==13.5.3
rpds-py==0.10.3
ruamel.yaml==0.17.32
ruamel.yaml.clib==0.2.7
s3fs==2023.9.2
s3transfer==0.6.2
scmrepo==1.3.1
shortuuid==1.0.11
shtab==1.6.4
six==1.16.0
smmap==5.0.1
sqltrie==0.7.0
stack-data==0.6.2
sympy==1.12
tabulate==0.9.0
tomli==2.0.1
tomlkit==0.12.1
torch==2.0.1
torchvision==0.15.2
tornado==6.3.3
tqdm==4.66.1
traitlets==5.10.0
triton==2.0.0
typing_extensions==4.8.0
tzdata==2023.3
urllib3==1.26.16
vine==5.0.0
voluptuous==0.13.1
wcwidth==0.2.6
wrapt==1.15.0
yarl==1.9.2
zc.lockfile==3.0.post1
zipp==3.17.0
</pre></details>

**Comment by furiosa-infra** [comment]
add3e67 nits: import early, packaging stuffs correctly
```

---------------------------------------------------------------------------------------------------------- benchmark: 9 tests ----------------------------------------------------------------------------------------------------------
Name (time in ms)                                                  Min                   Max                Mean              StdDev              Median                 IQR            Outliers       OPS            Rounds  Iterations
----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
test_mlcommons_resnet50_accuracy                                1.5226 (1.0)        522.3129 (11.06)      7.7945 (1.0)        5.9932 (1.63)       5.6116 (1.0)        3.0819 (2.63)    5627;6682  128.2956 (1.0)       50000           1
test_efficientnetb0_accuracy                                    4.9407 (3.24)       506.6570 (10.73)     13.1749 (1.69)       8.5187 (2.31)      12.3566 (2.20)       2.2876 (1.95)     908;2818   75.9019 (0.59)      50000           1
test_mlcommons_ssd_mobilenet_with_native_rust_pp_accuracy       5.6869 (3.73)        47.2054 (1.0)       12.2406 (1.57)       5.3866 (1.46)       9.4710 (1.69)       5.3932 (4.61)      805;328   81.6955 (0.64)       5000           1
test_mlcommons_ssd_mobilenet_accuracy                           7.1537 (4.70)       231.5826 (4.91)      11.3956 (1.46)       3.6834 (1.0)       10.7673 (1.92)       1.1703 (1.0)       523;776   87.7534 (0.68)       5000           1
test_efficientnetv2s_accuracy                                   7.4129 (4.87)       248.7642 (5.27)      14.9867 (1.92)       6.6797 (1.81)      14.3991 (2.57)       2.5243 (2.16)     948;1585   66.7260 (0.52)      50000           1
test_yolov5m_accuracy                                           9.1483 (6.01)       569.9067 (12.07)     17.1893 (2.21)      12.5290 (3.40)      13.6183 (2.43)       6.0402 (5.16)     839;1117   58.1757 (0.45)      10000           1
test_yolov5l_accuracy                                          12.0862 (7.94)       358.1844 (7.59)      16.6039 (2.13)       8.0943 (2.20)      15.3607 (2.74)       3.1162 (2.66)      143;319   60.2269 (0.47)      10000           1
test_mlcommons_ssd_resnet34_with_native_rust_pp_accuracy       39.2998 (25.81)    1,104.6358 (23.40)     50.6261 (6.50)      17.4529 (4.74)      50.4396 (8.99)       2.2129 (1.89)        6;443   19.7526 (0.15)       5000           1
test_mlcommons_ssd_resnet34_accuracy                          109.6047 (71.98)    1,522.3957 (32.25)    291.1836 (37.36)    111.6781 (30.32)    288.4513 (51.40)    105.2393 (89.92)     535;245    3.4343 (0.03)       5000           1
----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------

Legend:
  Outliers: 1 Standard Deviation from Mean; 1.5 IQR (InterQuartile Range) from 1st Quartile and 3rd Quartile.
  OPS: Operations Per Second, computed as 1 / Mean
```

File: docs/model_object.md [hunk 1]
```diff
@@ -40,19 +47,18 @@ To create a session, pass the `enf` field of the model object to the furiosa.run
...
 
-To utilize f32 source models, it is necessary to perform calibration and quantization.
-Pre-calibrated data is readily available in Furiosa-models, facilitating direct access to the quantization process.
-For manual quantization of the model, you can install the `furiosa-quantizer` package, which can be found at this  [package link](https://furiosa-ai.github.io/docs/latest/en/software/python-sdk.html#quantizer).
-The calib_range field of the model class represents this pre-calibrated data.
-After quantization, the output will be in the form of FuriosaAI's IR which can then be passed to the session.
-At this stage, the compiler configuration can be specified.
+To work with f32 source models, calibration and quantization are essential steps.
+You can access pre-calibrated data directly from furiosa-models, simplifying the quantization process.
+If you prefer manual quantization of the model, you can install the `furiosa-quantizer` package, available at this [package link](https://furiosa-ai.github.io/docs/latest/en/software/python-sdk.html#quantizer).
```
Comment (hyunsik) [inline docs/model_object.md:54]:
```suggestion
If you prefer a manual quantization step for a model, you can install the `furiosa-quantizer` package, available at this [package link](https://furiosa-ai.github.io/docs/latest/en/software/python-sdk.html#quantizer).
```

**Review by hyunsik — COMMENTED** [review]
[quantize_and_compile_model.ipynb](https://github.com/furiosa-ai/furiosa-models/pull/167/files#diff-0d4afab5928a671084ae4acd51969bf1b29d3c050eda6a8016a8f170a1aacfa3) seems to need a brief description of this example. It's hard to understand how the quantization and model zoo is related to each other. Also, it would be great if you add links to the [quantization section of SDK  doc](https://furiosa-ai.github.io/docs/latest/en/software/quantization.html).

File: docs/model_object.md (hunk 1 above)
Comment (furiosamg) [reply docs/model_object.md]: Applied in f771d98

File: docs/model_object.md [hunk 2]
```diff
@@ -1,37 +1,44 @@
...
-        show_source: true
+        show_root_heading: true
+        heading_level: 4
+        show_source: false
+
 
+## Inferencing with Runner API
 
-## Inferencing with Session API
+To create a Runner, pass the ENF binary obtained from the `model_source()` method of the model object to the `furiosa.runtime.sync.create_runner` function. If you prefer an asynchronous Runner, you can use the `furiosa.runtime.create_runner` function instead. Passing the pre-compiled ENF binary allows you to perform inference directly without the compilation process. Alternatively, you can also manually quantize and compile the original f32 model with the provided calibration range.
```
[the remark this answers is not in the document]
Comment (furiosamg) [reply docs/model_object.md]: Applied in f771d98

File: docs/model_object.md [hunk 3]
```diff
@@ -1,37 +1,44 @@
...
+
 ## Accessing artifacts and metadata
-A `Model` object includes model artifacts, such as ONNX, tflite, calibration range in yaml format, and ENF.
 
-ENF format is [FuriosaAI Compiler](https://furiosa-ai.github.io/docs/latest/en/software/compiler.html) specific format.
-Once you have the ENF file, you can reuse it to omit the compilation process that take up to minutes.
-In addition, a `Model` object has various metadata. The followings are all attributes belonging to a single `Model` object.
+A Model object encompasses model artifacts, such as ONNX, TFLite, mapping from a tensor name to the tensor's min and max, and ENF.
+
+The ENF format is specific to the FuriosaAI Compiler.
```
[the remark this answers is not in the document]
Comment (furiosamg) [reply docs/model_object.md]: Applied in f771d98

**Comment by furiosamg** [comment]
@hyunsik 리뷰 주신 부분들 반영했습니다. quantize & compile 문서도 더 적었습니다!

File: README.md [hunk 4]
```diff
@@ -9,20 +9,21 @@ for FuriosaAI NPU. However, all models are standard ONNX or tflite models,
...
 including programming guides, API references, and examples from the followings:
 
 * [Furiosa Models - Latest Documentation](https://furiosa-ai.github.io/furiosa-models/latest/)
-* [Model object](https://furiosa-ai.github.io/furiosa-models/latest/model_object/)
-* [Model List](https://furiosa-ai.github.io/furiosa-models/latest/#model_list)
-* [Command Tool](https://furiosa-ai.github.io/furiosa-models/latest/command_tool/)
-* [Furiosa SDK - Tutorial and Code Examples](https://furiosa-ai.github.io/docs/latest/en/software/tutorials.html)
+* [Model object](https://furiosa-ai.github.io/furiosa-models/v0.10.0/model_object/)
+* [Model List](https://furiosa-ai.github.io/furiosa-models/v0.10.0/#model_list)
+* [Command Tool](https://furiosa-ai.github.io/furiosa-models/v0.10.0/command_line_tool/)
```
Comment (libc-furiosa) [inline README.md:25]: 링크도 `Command Line Tool`로 변경해야할 것 같습니다.

```suggestion
* [Command Line Tool](https://furiosa-ai.github.io/furiosa-models/v0.10.0/command_line_tool/)
```

**Review by libc-furiosa — APPROVED** [review]
수고하셨습니다. 커멘트 하나 남겼는데 이것만 확인 부탁드립니다.

File: README.md (hunk 4 above)
Comment (furiosamg) [reply README.md]: 감사합니다 c1aca86a 에서 반영했습니다!

**Review by hyunsik — APPROVED** [review]
수고 많으셨습니다. publish 해도 좋을 것 같습니다.

**Pull request merged**